app/privacy-policy/page.tsx

```diff
@@ -1,183 +1,118 @@
-        We may disclose your personal information if required to do so by law or
-        if we believe that such action is necessary to (a) comply with a legal
-        obligation, (b) protect and defend our rights or property, (c) prevent
-        or investigate possible wrongdoing, or (d) protect the personal safety
-        of users or the public.
-      </p>
-
-      <h2>5. Your Rights</h2>
-      <p>
-        You have certain rights regarding the personal information we hold about
-        you. These rights may include:
-      </p>
-      <p>
-        &#8211; The right to access and obtain a copy of your personal
-        information
-        <br />
-        &#8211; The right to rectify any inaccurate or incomplete personal
-        information
-        <br />
-        &#8211; The right to erase your personal information under certain
-        circumstances
-        <br />
-        &#8211; The right to restrict the processing of your personal
-        information
-        <br />
-        &#8211; The right to object to the processing of your personal
-        information
-        <br />
-        &#8211; The right to data portability
-      </p>
-
-      <p>
-        To exercise your rights or make any privacy-related requests, please
-        contact us using the information provided in the &#8220;Contact
-        Us&#8221; section below.
-      </p>
-
-      <h2>6. International Data Transfers</h2>
-      <p>
-        Your personal information may be transferred to and processed in
-        countries outside your jurisdiction, including countries that may have
-        different data protection laws. We will take appropriate measures to
-        ensure that any international transfers comply with applicable data
-        protection laws.
-      </p>
-
-      <h2>7. Contact Us</h2>
-      <p>
-        If you have any questions, concerns, or requests regarding this Privacy
-        Policy or our privacy practices, please contact us:
-        hello@softwareconsultant.org
-      </p>
-      </div>
-    </>
+    <div
+      style={{
+        maxWidth: "1320x", /* Adjust max-width as needed */
+        margin: "0 auto", /* Center the container */
+        padding: "120px", /* Add padding */
+      }}>
+        
+      <Typography variant="h3"><b>{header1Details.title}</b></Typography><br/>
+      <Typography variant="subtitle1">
+        <b>{header1Details.effectiveDate}</b>
+      </Typography>
+      <Typography variant="body1">{header1Details.description}</Typography>
+      <br/>
+
+      <Typography variant="h5">{section1.title}</Typography><br/>
+      <Typography variant="subtitle1">{section1.subtitle}</Typography>
+
+      <ul style={{ listStyleType: "none", padding: 0 }}>
+        {section1.content.map((item, index) => (
+          <li key={index}>
+            <Typography variant="body1">{item}</Typography>
+          </li>
+        ))}
+      </ul>
+      
+
+      <Typography variant="subtitle1">{section102.subtitle}</Typography>
```

Comment:
Avoid using HTML tags such as <ul> and <li>. Instead use Material-UI equivalents such as <List> and <ListItem>.